mmv1/third_party/terraform/services/compute/resource_compute_subnetwork_test.go.tmpl

```diff
@@ -111,33 +111,35 @@ func TestAccComputeSubnetwork_update(t *testing.T) {
 			{
 				Config: testAccComputeSubnetwork_update1(cnName, "10.2.0.0/24", subnetworkName),
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckComputeSubnetworkExists(
-						t, "google_compute_subnetwork.network-with-private-google-access", &subnetwork),
+					testAccCheckComputeSubnetworkExists(t, "google_compute_subnetwork.network-with-private-google-access", &subnetwork),
 				),
 			},
 			{
 				// Expand IP CIDR range and update private_ip_google_access
 				Config: testAccComputeSubnetwork_update2(cnName, "10.2.0.0/16", subnetworkName),
-				Check: resource.ComposeTestCheckFunc(
-					testAccCheckComputeSubnetworkExists(
-						t, "google_compute_subnetwork.network-with-private-google-access", &subnetwork),
-				),
+				ConfigPlanChecks: resource.ConfigPlanChecks{
+                                    PreApply: []plancheck.PlanCheck{
+                                            plancheck.ExpectResourceAction("google_compute_subnetwork.network-with-private-google-access", plancheck.ResourceActionUpdate),
+                                    },
+                                },
 			},
 			{
```

Comment:
This step is failing because it modifies an immutable field - this is clearly supposed to be testing update-in-place logic but failed to do so because the ConfigPlanChecks weren't there previously. Please check whether the problem is that it's modifying a field it shouldn't, or whether there's a field it's modifying that is incorrectly (still?) immutable because of the same resource immutability issue.

Reply:
As discussed, not fixing this in this PR